src/main/res/values/strings.xml

```diff
@@ -42,6 +42,7 @@
     <string name="menu_settings">Settings</string>
     <string name="menu_share">Share</string>
     <string name="menu_delete">Delete</string>
+    <string name="menu_information">Information</string>
```

Comment:
"Properties" would be an apt word for this menu option as we are displaying file properties.

Reply:
I checked it with other apps. They are using "Information". Lets mentor give review on this one.